### PR TITLE
*: introduce write batch

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tirocks/src/write_batch.rs [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+    pub fn value(&self) -> &[u8] {
+        unsafe {
+            let mut buf = r(&[]);
+            tirocks_sys::crocksdb_writebatch_iterator_value(self.ptr, &mut buf);
+            s(buf)
+        }
+    }
+
+    #[inline]
+    pub fn value_type(&self) -> libc::c_char {
```
Comment (tabokie) [inline tirocks/src/write_batch.rs]: c_char?

Comment (BusyJay) [reply tirocks/src/write_batch.rs]: Yes, it's not public API unfortunately.

Comment (tabokie) [reply tirocks/src/write_batch.rs]: How about pub(crate)

**Review by tabokie — APPROVED** [review]

**Pull request merged**